typings/index.d.ts

```diff
@@ -509,7 +509,7 @@ declare module 'discord.js' {
     public setUsername(username: string): Promise<this>;
   }
 
-  export class Options {
+  export class Options extends null {
```

Comment:
To disallow constructing from TS:
```suggestion
  export class Options extends null {
    private constructor();
```